CONTRIBUTING.md

```diff
@@ -54,17 +54,127 @@ UIs for Backstage, Gitea, and ArgoCD are accessible on the machine:
 * Backstage: https://backstage.cnoe.localtest.me:8443/
 * ArgoCD: https://argocd.cnoe.localtest.me:8443/applications
 
-ArgoCD username is `admin` and the password can be obtained with 
-```
-kubectl -n argocd get secret argocd-initial-admin-secret -o go-template='{{ range $key, $value := .data }}{{ printf "%s: %s\n" $key ($value | base64decode) }}{{ end }}'
-```
+#### Getting credentials for packages
 
-Gitea admin credentials can be obtained with 
+Credentials for core packages can be obtained with: 
+```bash
+idpbuilder get secrets
 ```
-kubectl get secrets -n gitea gitea-admin-secret -o go-template='{{ range $key, $value := .data }}{{ printf "%s: %s\n" $key ($value | base64decode) }}{{ end }}'
+
+The `get secrets` command retrieves the following:
+- ArgoCD initial admin password.
+- Gitea admin user credentials.
+- Any secrets labeled with `cnoe.io/cli-secret=true`.
+
+You can think of the command as executing the following kubectl commands:
+
+```bash
+kubectl -n argocd get secret argocd-initial-admin-secret
+kubectl get secrets -n gitea gitea-admin-secret
+kubectl get secrets -A -l cnoe.io/cli-secret=true
```

Comment:
doesn't this belong to the main README? Independent of whether you want to contribute or not, this is still very handy for users.